services/web-server/src/login/strategies/mozilla-auth0.js

```diff
@@ -46,16 +45,16 @@ module.exports = class MozillaAuth0 {
         client_id: this.clientId,
         client_secret: this.clientSecret,
       });
-    const accessToken = JSON.parse(res.text).access_token;
+    const {
+      access_token: accessToken,
+      expires_in: expiresIn
+    }  = JSON.parse(res.text);
+    const expires = new Date().getTime() + (expiresIn * 1000);
```

Comment:
+1 although maybe not intended for this PR?

Reply:
This PR does both:
* Remove window.opener.location
* Stop decoding jwt